### PR TITLE
Fix ARTIFICIAL_CELL net_send.

### PR DESCRIPTION
**Pull request by 1uc**

### CODE REVIEW

File: test/usecases/net_send/simulate.py [hunk 1]
```diff
@@ -3,23 +3,51 @@
...
+    y = np.array(y_hoc.as_numpy())
+    art_y = np.array(art_y_hoc.as_numpy())
 
-assert np.all(np.abs(y - y_exact) == 0), f"{y} != {y_exact}, delta: {y - y_exact}"
+    return t, y, art_y
+
+
+def check_solution(t, y, art_y):
+    y_exact = np.array(t >= 2.0, np.float64)
+    art_y_exact = np.array(t >= 1.5, np.float64)
```
Comment (JCGoran) [inline test/usecases/net_send/simulate.py:33]: What do the two numbers here (2.0 and 1.5) represent? Are they related to some of the values used in the mod file?

Comment (1uc) [reply test/usecases/net_send/simulate.py:39]: I've added a brief description to the top. The two numbers are the time at which the event should be received. At that point `y` toggles "on".

Comment (JCGoran) [reply test/usecases/net_send/simulate.py:39]: Ahh, so the time is basically the first argument to `net_send` in the mod file?

**Review by JCGoran — APPROVED** [review]
LGTM overall, only minor comments.

File: test/usecases/net_send/simulate.py (hunk 1 above)
Comment (1uc) [reply test/usecases/net_send/simulate.py:39]: Correct, though technically, its the delay from the current time.

**Pull request merged**